daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/dfa/Rules.scala

```diff
@@ -488,9 +493,6 @@ class EECStateBlock(states: => ArrayBuffer[State], blockEnd: DFADelimiter, val E
     //
     // We've already encountered EEC as data0 here
     //
-    Rule { (r: Registers) => couldBeFirstChar(r.data0, blockEnd) } {
```

Comment:
The comment above on lines 489-492 is just misleading now. I suggest deleting it.